ops/ansible/kube/cfg-credentials.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: credentials
+type: Opaque
+data:
+  PG_PASSWORD: aWxvdmVib3RwcmVzcw==
```

Comment:
Left over?

Reply:
No. Default value if not specified from command line. This is base64 encoded password.
I guess I have a bug here. Will fix and notify you.